common/txmgr/txmgr.go

```diff
@@ -435,6 +435,19 @@ func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) Trigger(ad
 
 // CreateTransaction inserts a new transaction
 func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) CreateTransaction(txRequest txmgrtypes.TxRequest[ADDR, TX_HASH], qs ...pg.QOpt) (tx txmgrtypes.Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE], err error) {
+	// Check for existing Tx with RequestID. If found, return the Tx and do nothing
+	// Skipping CreateTransaction to avoid double send
+	if txRequest.RequestID != nil {
+		var etx *txmgrtypes.Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]
```

Comment:
nit: rename etx to existingTx or similar.
etx refers to old name, that implied Eth Tx, which no longer is correct, given this doesn't just represent ethereum/evm.